src/helpers.php

```diff
@@ -13,15 +13,16 @@
     function current_route(string $locale = null): string
     {
         $route = Route::getCurrentRoute();
-        $query = request()->getQueryString();
+        $parameters = request()->route()->parameters;
+        $query = request()->getQueryString() ?: [];
```

Comment:
No need the `?: [];` because it will be a string not an array, therefore we cannot use it with the `array_merge` anyways. So maybe something like: `$query = (array) request()->getQueryString();` to force an array?